docs/index.html

```diff
@@ -285,489 +141,465 @@
-        link: "haskell.html",
-        moved: -1
+        link: "swift.html",
+        moved: 0
       },
       {
         quadrant: 0,
-        ring: 1,
+        ring: 0,
         label: "Kotlin",
         active: true,
         link: "kotlin.html",
         moved: 0
       },
       {
         quadrant: 0,
-        ring: 1,
+        ring: 2,
         label: "TypeScript",
         active: true,
         link: "typescript.html",
         moved: 0
       },
       {
         quadrant: 0,
-        ring: 2,
-        label: "R",
+        ring: 3,
+        label: "Groovy",
         active: true,
-        link: "r.html",
+        link: "groovy.html",
         moved: 0
       },
       {
         quadrant: 0,
         ring: 3,
-        label: "Rust",
+        label: "TruffleRuby",
         active: true,
-        link: "rust.html",
-        moved: -1
-      },
-      {
-        quadrant: 3,
-        ring: 0,
-        label: "AWS SNS",
-        active: false,
-        link: "../queues/aws_sns.html",
+        link: "truffle_ruby.html",
         moved: 0
       },
       {
-        quadrant: 3,
-        ring: 0,
-        label: "AWS SQS",
-        active: false,
-        link: "../queues/aws_sqs.html",
+        quadrant: 0,
+        ring: 3,
+        label: "Crystal",
+        active: true,
+        link: "crystal.html",
         moved: 0
       },
       {
-        quadrant: 3,
-        ring: 0,
-        label: "Kafka",
-        active: false,
-        link: "../queues/kafka.html",
+        quadrant: 0,
+        ring: 3,
+        label: "Objective-C",
+        active: true,
+        link: "objective_c.html",
         moved: 0
       },
       {
-        quadrant: 3,
-        ring: 0,
-        label: "Nakadi",
-        active: false,
-        link: "../queues/nakadi.html",
+        quadrant: 0,
+        ring: 3,
+        label: "PHP",
+        active: true,
+        link: "php.html",
         moved: 0
       },
       {
-        quadrant: 3,
-        ring: 1,
-        label: "RabbitMQ",
-        active: false,
-        link: "../queues/rabbitmq.html",
+        quadrant: 0,
+        ring: 3,
+        label: "Ruby MRI",
+        active: true,
+        link: "mri.html",
         moved: 0
       },
       {
         quadrant: 3,
-        ring: 2,
-        label: "AWS Kinesis",
+        ring: 1,
+        label: "Azure Event Hub",
         active: false,
-        link: "../queues/aws_kinesis.html",
+        link: "../queues/eventhub.html",
         moved: 0
       },
       {
         quadrant: 3,
-        ring: 3,
-        label: "ActiveMQ",
+        ring: 0,
+        label: "RabbitMQ",
         active: false,
-        link: "../queues/activemq.html",
+        link: "../queues/rabbitmq.html",
         moved: 0
       },
       {
         quadrant: 3,
-        ring: 3,
-        label: "HornetQ",
+        ring: 1,
+        label: "Databricks",
         active: false,
-        link: "../queues/hornetq.html",
+        link: "../databricks.html",
```

Comment:
I haven't tried this actually but I assume these links would work if the target is available. Maybe at least align the paths somewhat and create the empty folders in the repo already (with just a .gitkeep file for now)

Reply:
I think we first need to agree if we want the detail pages also public. I guess that Zalando has their detail pages in a separate private repository and has a separate build for the internal version